go-controller/pkg/ovn/egressgw.go

```diff
@@ -108,6 +110,26 @@ func (oc *Controller) addGWRoutesForNamespace(namespace string, gws []net.IP, ns
 				if err != nil {
 					return fmt.Errorf("unable to add src-ip route to GR router, stderr:%q, err:%v", stderr, err)
 				}
+				if config.Gateway.Mode == config.GatewayModeHybrid {
+					// add allow policy to bypass lr-policy in GR
```

Comment:
Please please make this a function and just call it from all 3 places.

Reply:
done